src/js/index.js

```diff
@@ -0,0 +1,7 @@
+// All helper functions will be imported here, so that they can all be exported within one object.
+import { getBreakpoints, currentScreenWidth } from './modules/breakpointHelper';
+
+export {
+    getBreakpoints,
+    currentScreenWidth
```

Comment:
Would make this function name an action, so something like `getCurrentScreenWidth`